src/SevenSegmentTM1637.h

```diff
@@ -20,7 +20,13 @@
  #include <WProgram.h>
 #endif
 
-#include <avr/pgmspace.h>   // Used for PROGMEM
+//#include <avr/pgmspace.h>   // Used for PROGMEM
+// esp8266 compatibility
```

Comment:
It would make more sense to add this comment around line 28 where the ESP8266 compatible include occurs. That include will also work for other architectures so saying ESP8266 might be a bit overly specific.